docs/src/main/resources/microsite/_includes/_fetch-navigation.html

```diff
@@ -0,0 +1,32 @@
+<nav id="navigation">
+  <div class="wrapper">
+    <div class="nav-flex">
+        <a class="nav-brand" title="Fetch" href="{{site.data.commons.repoUrl}}">
+            <img src="img/navbar_brand.svg" title="{{ site.name }}">
+            <span>{{site.name}}</span>
+        </a>
+        <div class="nav-menu">
+          <button
+            class="button nav-icon-close"
+            title="Close"
+            onClick="toggleClass('.nav-menu', 'open');">
+              <img src="img/nav-icon-close.svg" alt="Close">
+          </button>
+            <ul>
+              {% for item in site.data.menu.nav%}
+              <li class="nav-menu-item">
+                  <a href={% if item.title == "Documentation" %}"{{site.url}}{{site.baseurl}}{{ item.url }}"{% else %}"{{ item.url }}"{% endif %} title="{{ item.title }}">{{ item.title }}</a>
```

Comment:
We need to review the link construction, see https://github.com/47deg/fetch/pull/203#issuecomment-529424948

Reply:
My bad, I was using the generated `jekyll` directory instead of `site` as I should be using.